app/src/org/commcare/android/database/user/models/FormRecordV5.java

```diff
@@ -0,0 +1,189 @@
+package org.commcare.android.database.user.models;
+
+import org.commcare.android.storage.framework.Persisted;
+import org.commcare.models.framework.Persisting;
+import org.commcare.modern.database.Table;
+import org.commcare.modern.models.EncryptedModel;
+import org.commcare.modern.models.MetaField;
+
+import java.util.Date;
+
+import static org.commcare.android.database.user.models.FormRecord.META_APP_ID;
+import static org.commcare.android.database.user.models.FormRecord.META_DISPLAY_NAME;
+import static org.commcare.android.database.user.models.FormRecord.META_FILE_PATH;
+import static org.commcare.android.database.user.models.FormRecord.META_LAST_MODIFIED;
+import static org.commcare.android.database.user.models.FormRecord.META_STATUS;
+import static org.commcare.android.database.user.models.FormRecord.META_SUBMISSION_ORDERING_NUMBER;
+import static org.commcare.android.database.user.models.FormRecord.META_UUID;
+import static org.commcare.android.database.user.models.FormRecord.META_XMLNS;
+import static org.commcare.android.database.user.models.FormRecord.QUARANTINE_REASON_AND_DETAIL_SEPARATOR;

```

Comment:
These should get copied into the old record rather than referenced. Same with all of the strings above.

Otherwise a new version of the `FormRecord` class in the future could change the static String and affect the deserialization of this class.